src/InputParser.hpp

```diff
@@ -12,7 +12,7 @@
 // Globals
 #define BASE_TEN 10
 static const std::string progname = "LINKS";
-static const std::string version = "2.0.0";
+static const std::string version = "2.0.1";
```

Comment:
```suggestion
static const std::string version = PACKAGE_VERSION;
```
You can use the version provided by automake. You'd need to `#include "config.h"` though.